authors/settings.py

```diff
@@ -144,3 +146,4 @@
         # 'authors.apps.authentication.backends.JWTAuthentication',
     ),
 }
+STATICFILES_STORAGE = 'whitenoise.storage.CompressedManifestStaticFilesStorage'
```

Comment:
Consider adding this line of code which helps to access files in the static folder like the CSS
STATIC_ROOT = os.path.join(BASE_DIR, 'static/')

Reply:
@eduhmikI have handled this using django_heroku